service/src/main/java/com/commercetools/pspadapter/payone/IntegrationService.java

```diff
@@ -53,6 +56,19 @@ public void start() {
 
         Spark.port(port());
 
+        redirect.any("/", "/health", Redirect.Status.MOVED_PERMANENTLY);
```

Comment:
i think it is confusing to redirect base URL always to health as it should return 404 instead